cesium/delete.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+package cesium
+
+import (
+	"github.com/cockroachdb/errors"
+	"strconv"
+)
+
+func (db *DB) DeleteChannel(ch ChannelKey) error {
+	db.mu.Lock()
+	defer db.mu.Unlock()
+	udb, uok := db.unaryDBs[ch]
+	if uok {
+		if err := udb.TryClose(); err == nil {
```

Comment:
You dont need an else block here because you're doing an early return. The way I typically do it is try to keep the normal or 'happy' path as un-nested as possible. I would swap this case around, check for a non-nil error and return early if you find one.